### DISCUSSION TITLE
z-index conflict for hero-content and menu in navbar.

### DISCUSSION THREAD

**Discussion by prudhvi-mallavarapu** [saadeghi/daisyui#2245] [Q&A] [answered]
I'm using latest "tailwindcss": "^3.3.3" and "daisyui": "^3.5.1".  

Hero sections https://daisyui.com/components/hero/ and Navbar https://daisyui.com/components/navbar/responsive (dropdown menu on small screen, center menu on large screen) submenu items are displayed below level to the hero-content.

Check the tailwindplay demo code and output here.
https://play.tailwindcss.com/Sbt2rV4HJU

![image](https://github.com/saadeghi/daisyui/assets/25653002/c227528d-35c7-4f32-9f5d-60eae1ff56f8)

**Comment by prudhvi-mallavarapu** [comment, marked as answer]
adding `z-[1]` class solved issue for `<ul className="menu menu-horizontal px-1"></ul>`

**Reply by kisamoto** [reply]
Does this still work for you? I have a `z-[1]` also on the menu but still getting overlap with hero text.
